### PR TITLE
✨ add link on cases column

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: modules/node_modules/@ncigdc/containers/FileTable.js [hunk 1]
```diff
@@ -139,6 +139,7 @@ export const FileTableQuery = {
                 total
```
Comment (shanewilson) [inline modules/node_modules/@ncigdc/containers/FileTable.js]: I would only ask for 1 case

**Comment by shanewilson** [comment]
I don't think we need the approved tag - since there are reviews built in now

File: modules/node_modules/@ncigdc/containers/FileTable.js (hunk 1 above)
Comment (shanewilson) [inline modules/node_modules/@ncigdc/containers/FileTable.js]: I would only ask for 1 case since you only need the actual data from the first hit

Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/FileTable.js]: It's using the full array in at least one spot 
https://github.com/NCI-GDC/portal-ui/blob/a50bfe6d3069373d5da9a1165979abaada7318a5/modules/node_modules/%40ncigdc/containers/FileTr.js#L72

Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/FileTable.js]: makes sense, updated 👍

**Pull request merged**